arch/SPECS/wazuh-agent/arch/wazuh.install

```diff
@@ -67,7 +67,7 @@ post_upgrade() {
 }
 
 post_install() {
-    if ! getent group | grep -q "^ossec" ; then
+    if ! getent group | grep -q "^wazuh" ; then
```

Comment:
Please, change all references to wazuh user or group with the corresponding variables:

- `${USER}` 
- `${GROUP}`